### DISCUSSION TITLE
Any idea why shadow-root is not always properly loaded in Linux-Safari?

### DISCUSSION THREAD

**Discussion by fdm-mateco** [stenciljs/core#5945] [Stencil Testing]
We have a Playwright test suite which is now being extended to run across Chromium, Firefox, Edge, Chrome and Safari.

Running against Safari does pose drastic issues, most notably the shadow-root being empty (hence the component being invisible).
![image](https://github.com/user-attachments/assets/858e6278-3cd7-49e5-afaf-8f4b94e5bd59)

Unfortunately, it does not occur when I run this test locally on Windows-Safari, but often (though not always!) on Playwright's official Docker. Any input appreciated.

**Comment by christian-bromann** [comment]
Hey @fdm-mateco 👋  thanks for raising this discussion. Unfortunately there is nothing we can do about this as we don't know whether this is related to Stencil or to the fact that you are using Playwright browser which come with modifications and can introduce bugs like this. I personally would not recommend to use Playwright for this exact reason. If you can reproduce this on a real browser, I would be happy to take a look a this.
